src/java/com/github/jsdossier/CompilerModule.java

```diff
@@ -78,7 +79,7 @@ CompilerOptions provideCompilerOptions(
 
     // IDE mode must be enabled or all of the jsdoc info will be stripped from the AST.
     options.setIdeMode(true);
-    options.setPreserveJsDocWhitespace(true);
+    options.setParseJsDocDocumentation(Config.JsDocParsing.INCLUDE_DESCRIPTIONS_WITH_WHITESPACE);
```

Comment:
Guess I could go back to a snapshot (e5226857d30a13aa1c4ff172913ddb04d3d1b304) - or are there compiler snapshots on sonatype (already have a snapshot dep for [guava](https://github.com/jleyba/js-dossier/blob/master/WORKSPACE#L61))?

On a more general note, does the compiler have a deprecation policy for public APIs? I could see this causing headaches for anyone else building extensions for the compiler